injector.js

```diff
@@ -363,18 +363,18 @@ Injector.Util.getContext = function(contextName) {
  * @param  {Function} requirejs RequireJS instance
  * @return {Injector}           Injector function
  */
-Injector.provide = function(requirejs) {
+Injector.provide = function (requirejs) {
   Injector.requirejs = requirejs;
   return Injector;
 };
 
-Injector._ensureRequireJS = function() {
-  if (typeof Injector.requirejs !== 'function') {
+Injector._ensureRequireJS = function () {
+  if (!_.isFunction(Injector.requirejs())) {
```

Comment:
That is wrong. It should not be invoked.

```suggestion
  if (!_.isFunction(Injector.requirejs)) {
```